### DISCUSSION TITLE
Data for word_count benchmark

### DISCUSSION THREAD

**Discussion by DamjanLasicJurkovic** [exaloop/codon#311] [General]
Hi,

congrats, very nice effort on the lib. I'd be interested in running some of the benchmarks locally, but I can't seem to find the input data for word_count in the repo. Could anyone please provide it or point me to its location?

Best regards,
Damjan

**Comment by arshajii** [comment]
Hi @DamjanLasicJurkovic -- here is the file we used for that benchmark: https://www.dropbox.com/s/dvj6zrqeo30tnzt/tweets.txt?dl=0 (~170MB -- it's just a dump of financial tweets). Let me know if you have any trouble accessing/downloading it!

**Reply by DamjanLasicJurkovic** [reply]
Hi, thanks for providing the file!
